src/routes/directions.js

```diff
@@ -1,9 +1,10 @@
 const express = require('express');
 const router = express.Router();
-
+require('dotenv').config()
```

Comment:
@johanbmk you'll need a `;` at the end of this line. Please make sure to run the linter before committing, thanks!